ios/RCTTWVideoModule.m

```diff
@@ -96,18 +98,302 @@ - (void)addParticipantView:(TVIVideoView *)view identity:(NSString *)identity  t
+                           if([audioTrack roundTripTime]){
+                               NSNumber *roundTripTime= [f  numberFromString:[NSString stringWithFormat:@"%llu", [audioTrack roundTripTime]]];
+                               [outgoingTrack setObject:roundTripTime forKey:@"roundTripTime"];
+                           }
+                           if([audioTrack audioLevel]){
+                               NSNumber *audioLevel= [f  numberFromString:[NSString stringWithFormat:@"%lu", (unsigned long)[audioTrack audioLevel]]];
+                               [outgoingTrack setObject:audioLevel forKey:@"audioLevel"];
+                           }
+                           if([audioTrack jitter]){
+                               NSNumber *jitter= [f  numberFromString:[NSString stringWithFormat:@"%lu", [audioTrack jitter]]];
+                               [outgoingTrack setObject:jitter forKey:@"jitter"];
+                           }
+                           if([audioTrack codec]){
+                               NSString *codec = [NSString stringWithFormat:@"%@", [audioTrack codec]];
+                               [outgoingTrack setObject:codec forKey:@"codec"];
+                           }
 
-RCT_EXPORT_METHOD(startLocalVideo:(BOOL)screenShare) {
+                           if([audioTrack ssrc]){
+                               NSNumber *ssrc= [f  numberFromString:[NSString stringWithFormat:@"%lu", [audioTrack jitter]]];
+                               [outgoingTrack setObject:ssrc forKey:@"ssrc"];
+                           }
+                           if([audioTrack trackId]){
+                               trackId = [NSString stringWithFormat:@"%@", [audioTrack trackId]];
+                               [outgoingTrack setObject:trackId forKey:@"trackId"];
+                           }
+
+                           if([audioTrack packetsSent]){
+                               NSNumber *packetsSent= [f  numberFromString:[NSString stringWithFormat:@"%lu", [audioTrack packetsSent]]];
+                               [outgoingTrack setObject:packetsSent forKey:@"packetsSent"];
+                           }
+
+                           if([audioTrack packetsLost]){
+                               NSNumber *packetsLost= [f  numberFromString:[NSString stringWithFormat:@"%lu", [audioTrack packetsLost]]];
+                               [outgoingTrack setObject:packetsLost forKey:@"packetsLost"];
+                           }
+                           if([audioTrack valueForKey:@"timestamp"]){
+                               NSNumber *timestamp= [f  numberFromString:[NSString stringWithFormat:@"%lu", (unsigned long)[audioTrack valueForKey:@"timestamp"]]];
+                               [outgoingTrack setObject:timestamp forKey:@"timestamp"];
+                           }
+
+                           [localAudioTracks setObject:outgoingTrack forKey:trackId];
+
+
+                       }
+                   }
+
+
+
```

Comment:
Not sure if it's just github but there seem to be some spacing issues

Reply:
needs a formating check for sure.